uniffi_bindgen/src/interface/attributes.rs

```diff
@@ -637,15 +642,4 @@ mod test {
             "conflicting attributes on interface definition"
         );
     }
-
-    #[test]
-    fn test_other_attributes_not_supported_for_interfaces() {
-        let (_, node) =
-            weedle::attribute::ExtendedAttributeList::parse("[Threadsafe, Error]").unwrap();
```

Comment:
Could we keep this test, but use some other attribute like `ByRef` or similar to replace `Error`?